### ISSUE TITLE
Madplay error

### ISSUE THREAD

**Issue by funful** [redchenjs/mt76x8-wm8960#9]
Hi Jack Chen 
I have a problem.
```
root@OpenWrt:~#
root@OpenWrt:~#
root@OpenWrt:~# cd /root/
root@OpenWrt:~# madplay gamestartup.mp3
MPEG Audio Decoder 0.15.2 (beta) - Copyright (C) 2000-2004 Robert Leslie et al.
ALSA lib pcm_hw.c:1829:(_snd_pcm_hw_open) Invalid value for card
audio: No such file or directory
root@OpenWrt:~#
root@OpenWrt:~#
root@OpenWrt:~#

```

**Comment by redchenjs** [comment]
Hi @funful,

Which OpenWrt / Kernel version are you using? `madplay` works fine on my `LinkIt Smart 7688` board with `OpenWrt r16595 / Kernel 5.4.113`

**Comment by redchenjs** [comment]
Please note that the `master` branch and the `linux-5.4` branch have different dts files, they are not compatible.

**Comment by funful** [comment]
OK, let me try again. I'm using Kernel 5.4.113. Device type 7628

**Comment by funful** [comment]
Hi， @redchenjs  Which Openwrt version can I use?

**Comment by redchenjs** [comment]
You can use `openwrt-21.02` or `master` branch, example dts files: [linux-5.4/examples](https://github.com/redchenjs/mt762x-wm8960/tree/linux-5.4/examples)
